beancount.el

```diff
@@ -425,6 +425,20 @@ With an argument move to the next non cleared transaction."
         (setq done t)))
     (if (not done) (goto-char (point-max)))))
 
+(defun beancount-goto-previous-transaction (&optional arg)
+  "Move to the previous transaction.
+With an argument move to the next non cleared transaction."
```

Comment:
`s/next/previous/` in the docstring. Also this is broken when an argument is specified.

Reply:
Fixed the docstring and jumping to previous non-cleared transaction on prefix arg